library/network/src/lib/y2firewall/firewalld/api/services.rb

```diff
@@ -29,6 +29,9 @@ class Api
       # definition and configuration.
       module Services
         # @param service [String] The firewall service
+        # @param permanent [
+        # @param permanent [Boolean] if true it adds the --permanent option the
+        # command to be executed
```

Comment:
now it looks a bit messy - a copy paste issue?

Reply:
ouch, yep, that issue